test/index.js

```diff
@@ -152,6 +153,26 @@ describe('KISSmetrics', function () {
       result['Billing Amount'].should.eql(19.95);
       result._n.should.eql('Baked a cake');
     });
+
+    it('should use a POST request if the payload exceeds a threshold', function(done){
+      var largeEvent = helpers.repeat('x', 2100);
+      var timestamp = Math.floor(Date.now() / 1000);
+
+      test
+        .set(settings)
+        .track({event: largeEvent, time: timestamp})
```

Comment:
> Most people don't send a very large event name -- usually it would be because they attached too many properties or traits.

Both cases lead to a large query string payload which is what we're trying to reproduce.

> Can we also get a unit test to make sure it does not use a POST request if the payload does not exceed the threshold?

See e3e456a25dfe20781480d0c3b1832ed21b63cbea

> Also nit pick on line 163-- can we use the fixture convention like you see in the other unit tests?

My only reasoning against putting the large payload in a JSON file is that it's not clear that a >2KB string is >2KB. I think it's clearer generating the string from within the test function.